srt.go

```diff
@@ -127,24 +133,18 @@ func ReadFromSRT(i io.Reader) (o *Subtitles, err error) {
 }
 
 // parseTextSrt parses the input line to fill the Line
-func parseTextSrt(i string) (o Line) {
+func parseTextSrt(i string, styles StyleAttributes) (Line, StyleAttributes) {
```

Comment:
Could you change the function signature to `func parseTextSrt(i string, sa *StyleAttributes) (o Line)` instead? That way you can update the pointer directly 👍